Library/Formula/makemake.rb

```diff
@@ -0,0 +1,15 @@
+class Makemake < Formula
+  homepage "https://github.com/Julow/makemake"
+  url "https://github.com/Julow/makemake.git", :tag => "1.0-release"
```

Comment:
Please specify a `:revision` along with the tag, which prevents retagging issues.